azurerm/resource_arm_data_lake_store_test.go

```diff
@@ -0,0 +1,208 @@
+package azurerm
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAzureRMDataLakeStore_payasyougo(t *testing.T) {
+	resourceName := "azurerm_data_lake_store.test"
+	ri := acctest.RandIntRange(1, 999999)
+	config := testAccAzureRMDataLakeStore_payasyougo(ri, testLocation())
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMDataLakeStoreDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: config,
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMDataLakeStoreExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "tier", "Consumption"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAzureRMDataLakeStore_monthlycommitment(t *testing.T) {
```

Comment:
Could we change this to `TestAccAzureRMDataLakeStore_tier`?

Reply:
done